basics/basic-types.md

```diff
@@ -23,7 +23,7 @@ om programmet är kompilerat för ett 32-bitar eller ett 64-bitars system.
 |`real`   | >= 64-bit (generellt 64-bit, men 80-bit på Intel x86 32-bit)
 
 Prefixet `u` betecknar en *osignerad* typ. `char` översätter till
-UTF-8 tecken, `wchar` används i UTF-16 strängar och `dchar`
+ett UTF-8 tecken, `wchar` används i UTF-16 strängar och `dchar`
```

Comment:
I am not entirely sure of how to translate it properly then, the english paragraph reads:

> The prefix u denotes unsigned types. char translates to UTF-8 characters, wchar is used in UTF-16 strings and dchar in UTF-32 strings.

Do you have any suggestions?